### PR TITLE
CC-remove-API-dns

### PR DESCRIPTION
**Pull request by bmckercher123**
### Remove references to Elasticsearch API for CC docs (not applicable)

- For 3.15 and next
- Change title to reflect task, no other edits
- Request from Erik

### CODE REVIEW

File: calico-cloud/visibility/elastic/dns/dns-logs.mdx [hunk 1]
```diff
@@ -55,58 +55,11 @@ The `latency_*` fields provide information about the latency of the DNS lookups
...
-and `servers` objects. Querying on a nested object field is not supported by Kibana, but can be
-done with the Elasticsearch API, for example:
+with a query on the qname field, you could find all of the DNS response information that was provided to clients trying to resolve a particular domain name
 
-```shell
-curl 'http://10.111.1.235:9200/tigera_secure_ee_dns.cluster.20190711/_search?pretty=true' \
-    -d '{"size": 2, "query": {"nested": {"path": "rrsets", "query": {"match": {"rrsets.rdata": "18.103.110.45"}}}}}' \
-    -H "Content-Type: application/json"
-```
+with a query on the rrsets.rdata field, you could find all of the DNS lookups that included a particular IP address in their response data.
```
Comment (ctauchen) [inline calico-cloud/visibility/elastic/dns/dns-logs.mdx]:
```suggestion
with a query on the `rrsets.rdata` field, you could find all of the DNS lookups that included a particular IP address in their response data.
```

Yes? If yes, in both docs.

File: calico-cloud/visibility/elastic/dns/dns-logs.mdx [hunk 2]
```diff
@@ -55,58 +55,11 @@ The `latency_*` fields provide information about the latency of the DNS lookups
...
-Note that that query can also be written with a JSON body, like this:
-
-```shell
-curl 'http://10.111.1.235:9200/tigera_secure_ee_dns.cluster.20190711/_search?pretty=true' \
-    -d '{"query": {"match": {"qname": "example.com"}}}' \
-    -H "Content-Type: application/json"
-```
-
-### Querying on a nested object field
+Once a set of DNS logs has accumulated in Elasticsearch, you can perform many interesting queries. For example:
```
Comment (ctauchen) [inline calico-cloud/visibility/elastic/dns/dns-logs.mdx]: I wonder whether the following two sentences would make more sense as an unordered list.

**Review by ctauchen — DISMISSED** [review]
LGTM, pending suggestions

File: calico-cloud/visibility/elastic/dns/dns-logs.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud/visibility/elastic/dns/dns-logs.mdx]: Fixed.

File: calico-cloud/visibility/elastic/dns/dns-logs.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud/visibility/elastic/dns/dns-logs.mdx]: Sure. Bullets work nicely.

**Pull request merged**